### PR TITLE
connectors: removed note from Iceberg connector doc, added converters doc draft

### PR DESCRIPTION
**Pull request by bochenekmartin**

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+To use the converter, select `String` option as a key or value message format.
+
+== JSON Converter
+
+JSON Converter can be specified by selecting `JSON` option as a key or value message format.
+The connector needs a message schema to check message format. Usually, the schema is stored in Schema Registry. 
+
+However, if the Schema Registry cannot be used, embedded JSON schema is an option. Then each message contains a schema. It will result in a large size of a message. Example JSON message with embedded schema:
+[source,json]
```
Comment (tomasz-sadura) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: Can we make it more clear that for JSON Converter we only support schema embedded in the message?
"Usually, the schema is stored in Schema Registry" and "embedded JSON schema is an option" may be misleading.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+== CloudEvents Converter
+
+CloudEvents Converter is specific for Debezium PostgreSQL and MySQL source connectors. 
+
+See also: https://debezium.io/documentation/reference/2.2/integrations/cloudevents.html[CloudEvents Converter official documentation]
+
+== Protobuf Converter {badge-cloud-beta}
+
+Protobuf Converter requires schema in Schema Registry.
+Currently the converter supports primitive types in sink connectors only.
```
Comment (tomasz-sadura) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: Do we support all primitive types, and if not could we list limitations?

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+:description: Connectors Converters and serialization
+:page-cloud: true
+
+== Key and value converters
+
+Each Redpanda message is a key and value record. Converters handle serialization and deserialization of data between Redpanda topic and a connector. Record key and value converters are configured separately. It can be specified using `Redpanda message key format` and `Redpanda message value format` properties. Key and value converters can be different.
+
+[NOTE]
+====
+If an external system requires structured data (like BigQuery, Snowflake, SQL database), you need to provide data with schema. Then you need to use Avro Converter, Protobuf Converter or JSON Converter with schema.
```
Comment (tomasz-sadura) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: Snowflake is a little bit tricky. I think it does not require the schema actually.

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc (hunk 2 above)
Comment (bochenekmartin) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc]: Good point, I will make it more specific shortly

**Comment by Feediver1** [comment]
@bochenekmartin Is this ready for doc review? Or do you expect additional content/updates?

**Comment by bochenekmartin** [comment]
> @bochenekmartin Is this ready for doc review? Or do you expect additional content/updates?

@Feediver1 , it is ready for review 👍

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 4]
```diff
@@ -4,8 +4,6 @@
 :page-cloud: true
 :page-beta: true
 
-include::shared:partial$feature-flag.adoc[]
-
 You can use the Iceberg Sink connector to write data into Iceberg tables.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
You can use the Iceberg Sink connector to accomplish the following: 

* Write data into Iceberg tables
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 5]
```diff
@@ -4,8 +4,6 @@
...
 
 == Features
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
```

**Comment by micheleRP** [comment]
@bochenekmartin and @tomasz-sadura: I'm making some minor style edits. I've also rearranged the order in the left nav tree/index page. Please [check this](https://deploy-preview-131--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/managed-connectors/) to confirm if you agree with this order!

**Comment by Feediver1** [comment]
> @bochenekmartin and @tomasz-sadura: I'm making some minor style edits. I've also rearranged the order in the left nav tree/index page. Please [check this](https://deploy-preview-131--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/managed-connectors/) to confirm if you agree with this order!

I think this is a great update--keeps these two new sections separate from all of the actual connectors. Then, later when Cloud docs go into a new branch in docs, these two topics can go under Manage.

**Review by micheleRP — APPROVED** [review]
Thanks @bochenekmartin! I know you're still working this but doc edits are complete and you can merge this when you're ready. Let me know if you want me to review it again after you finish your updates.

**Pull request merged**